sdk/storage/azblob/blob/client.go

```diff
@@ -301,10 +268,10 @@ func (b *Client) GetSASToken(permissions SASPermissions, start time.Time, expiry
 
 // Concurrent Download Functions -----------------------------------------------------------------------------------------
 
-// DownloadToWriterAt downloads an Azure blob to a WriterAt in parallel.
-func (b *Client) DownloadToWriterAt(ctx context.Context, writer io.WriterAt, o *DownloadToWriterAtOptions) error {
+// downloadWriterAt downloads an Azure blob to a WriterAt in parallel.
+func (b *Client) downloadWriterAt(ctx context.Context, writer io.WriterAt, o *DownloadWriterAtOptions) error {
```

Comment:
Do we need to export `DownloadWriterAtOptions`?

Reply:
Done. Not required to export to made it lowercase.